accounts/models.py

```diff
@@ -6,6 +6,42 @@
 from phonenumber_field.modelfields import PhoneNumberField
 
 
+class School(models.Model):
+    """
+    Represents a school (ex: School of Arts & Sciences).
+    """
+
+    # boolean field for whether major is currently active/inactive
+    name = models.TextField(primary_key=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Major(models.Model):
+    """
+    Represents a Major (ex: Accounting, BS).
+    """
+
+    name = models.TextField(primary_key=True)
+    # boolean field for whether major is currently active/inactive
+    is_active = models.BooleanField(default=True)
+
+    DEGREE_BACHELOR = "Bachelor's"
```

Comment:
The strings for `DEGREE_*` should be something like `BACHELORS` since that will go into the DB itself

Reply:
fixed